ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -25,21 +25,30 @@
     private final PartyMapper partyMapper;
     private final TheirDetailsMapper theirDetailsMapper;
     private final AmountMapper amountMapper;
+    private final PaymentMapper paymentMapper;
+    private final InterestMapper interestMapper;
+    private final InterestDateMapper interestDateMapper;
 
     @Autowired
     public ClaimMapper(PersonalInjuryMapper personalInjuryMapper,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS00107)